src/pages/CreateAccount/CreateAccount.tsx

```diff
@@ -37,98 +43,109 @@ export const CreateAccount = () => {
       exit={{ opacity: 0 }}
       style={{ height: 'inherit' }}
     >
-      <Centralizer>
-        <AutoCenter>
-          <Styled.Title>Boardy</Styled.Title>
-        </AutoCenter>
-        <Styled.Paragraph>
-          Please fill your details to create your account.
-        </Styled.Paragraph>
-        <Form name="normal_login" layout="vertical" onFinish={createNewAccount}>
-          <Form.Item
-            name={'firstName'}
-            label="First name"
-            rules={[
-              {
-                required: true,
-                message: 'Please input your first name!',
-              },
-            ]}
+      <Container
+        sx={{
+          height: 'inherit',
```

Comment:
Yep, you're right, I have checked, and we have some props to handle the CSS, mainly in the Box component (that can be used for everything because you can define what HTML component it'll be, you do it using the props component, like this: component="form"). 

With this, I changed where it was possible to use the MUI props instead sx.